ygen/yang_helpers.go

```diff
@@ -253,3 +253,20 @@ func appendIfNotEmpty(slice []string, s string) []string {
 	}
 	return slice
 }
+
+// appendEntriesNotIn appends strings that are in the ne slice but not the ee
+// slice to the ee slice and returns it.
+func appendEntriesNotIn(ee []string, ne []string) []string {
```

Comment:
Per comment above, would suggest deleting this and just using a map directly. Otherwise consider re-implementing as a map.

Reply:
Reimplemented.